src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
+
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_VALUE = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int RESIZE_INDEX = 2;
+    private Node [] elements = new Node[DEFAULT_VALUE];
+    private int size;
+
+    private class Node<K,V> {
```

Comment:
private classes too should go after public methods

Reply:
Replace class